### PR TITLE
Feat: Add .bat file to automate cloning the repository

### PR DESCRIPTION
**Pull request by Prashanth-Chandra**
The bat file will automatically clone the repository and opens the index.html in it

https://github.com/SrNightmare09/chess-fen-reader/assets/91191641/c53142b5-a7b2-4069-a7c8-4475f6f5f29b

### CODE REVIEW

File: server/script.bat [hunk 1]
```diff
@@ -1,13 +1,18 @@
 @echo off
```
Comment (SrNightmare09) [inline server/script.bat:2]: Please remove any extra whitespace which might be present in the file.

File: server/script.bat [hunk 2]
```diff
@@ -1,13 +1,18 @@
...
-git clone
+set DIR_NAME="%CD%\chess-fen-reader"
 
-start "" "%PARENT_DIR%"
\ No newline at end of file
+git clone %REPO_URL%
+    
+cd %CLONE_DIR%\chess-fen-reader\src\
+
+echo Current directory: %CD%
```
Comment (SrNightmare09) [inline server/script.bat:14]: I don't believe there is a requirement to print the current directory in the terminal. If there is, please feel free to reply to this change request.

**Review by SrNightmare09 — COMMENTED** [review]
Wow! It really works like magic! I have requested two minor changes, after which we can merge your PR!

**Comment by Prashanth-Chandra** [comment]
@SrNightmare09 can you review it now

**Comment by SrNightmare09** [comment]
@Prashanth-Chandra looks good! merged 🔨

**Pull request merged**